### DISCUSSION TITLE
what to do

### DISCUSSION THREAD

**Discussion by constanceee550** [mastodon/mastodon#23589] [General]
I don't know what to do. I am here, now what. I can not remember the instance I joined, how do I find it?  how do I use Mastodon?  How do I put my photo/art on here with a link back to the website my pictures are on? I will be grateful for a simple answer to my questions.

**Comment by mjankowski** [comment]
I am also here, and like you I dont know what to do.

That said, if you forget your instance ... I'm not sure how to help. Check your email for notifications you got on signup?
